scripts/exglobal_atmos_products.sh

```diff
@@ -204,13 +216,13 @@ if [[ "${SENDDBN:-}" == "YES" ]]; then
   if [[ "${RUN}" == "gfs" ]]; then
     "${DBNROOT}/bin/dbn_alert" MODEL "${RUN^^}_PGB2B_0P25"      "${job}" "${COM_ATMOS_GRIB_0p25}/${PREFIX}pgrb2b.0p25.${fhr3}"
     "${DBNROOT}/bin/dbn_alert" MODEL "${RUN^^}_PGB2B_0P25_WIDX" "${job}" "${COM_ATMOS_GRIB_0p25}/${PREFIX}pgrb2b.0p25.${fhr3}.idx"
```

Comment:
@aerorahul Could you also add a dbnet alert for wgne dataset from
https://github.com/NOAA-EMC/UPP/blob/release/gfs_v16/scripts/exgfs_atmos_nceppost.sh#L400

Reply:
yes. will do.  thanks

Reply:
done.